admin/timescale/updates/2021-08-03-after-migration-listens-add-user-id.sql

```diff
@@ -0,0 +1,9 @@
+-- index so that we can quickly see which listens have user_id of 0. We'll need it when selecting
+-- on this field after we remove user_name, too.
+-- It doesn't appear to affect bulk update speed too much with this index in place
+-- CREATE INDEX will lock the whole table, which means that it'll be unavailable while the index
+-- is building. Timescale doesn't support CREATE INDEX CONCURRENTLY. The best we can do is this
+-- extension which opens a transaction and creates the index one chunk at a time.
+-- This lets inserts and selects work on the table while it's running
+-- This operation doesn't support being run inside a transaction.
+create index user_id_listen on listen(user_id) WITH (timescaledb.transaction_per_chunk);
```

Comment:
I created this specific index only to be able to check which listen rows hadn't been updated when restarting the update script if it crashed (see the comment in line 1).
We'll need an index on user_id in order to select listens, but that'll probably be an index over a few different columns - see the current indexes on the listen table that use user_name, they're all combined with other columns:
```
Indexes:
    "listened_at_track_name_user_name_ndx_listen" UNIQUE, btree (listened_at DESC, track_name, user_name)
    "listen_listened_at_idx" btree (listened_at DESC)
    "listened_at_user_name_ndx_listen" btree (listened_at DESC, user_name)
```

Therefore I think we can get rid of this file, and add the index in part 2 of the issue